convert.py

```diff
@@ -84,7 +84,7 @@ def convert(args):
                     # TODO(kuke): deal with the corner case that vars in 
                     #     different blocks have the same name
                     node_proto = ops.node_maker[op.type](operator=op,
-                                                         scope=inference_scope)
+                                                         block=block)
```

Comment:
In `batch_norm_op`, we need to know the shape of input data, which is non-persistable, so here `block` is needed.